<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIPanel</class>
 <widget class="QWidget" name="GUIPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUIPanel</string>
  </property>
  <widget class="QPushButton" name="pingButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>0</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ping</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="hostAddress">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>localhost</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stateButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>State:</string>
   </property>
  </widget>
  <widget class="QLabel" name="statusLabel">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>620</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stoped</string>
   </property>
  </widget>
  <widget class="QLabel" name="hostLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MQTT Broker: </string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>0</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>981</width>
     <height>531</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="generalTab">
    <attribute name="title">
     <string>General</string>
    </attribute>
    <widget class="QGroupBox" name="groupAmbient">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>451</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Ambient Conditions</string>
     </property>
     <widget class="ThermoMeter" name="temperature">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="toolTip">
       <string>Shows the temperature</string>
      </property>
      <property name="whatsThis">
       <string>The bar meter widget displays the pressure attached to it</string>
      </property>
      <property name="minimum">
       <double>5.000000000000000</double>
      </property>
      <property name="maximum">
       <double>60.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
      <property name="suffix">
       <string> [ÂºC]</string>
      </property>
      <property name="valueOffset">
       <double>275.000000000000000</double>
      </property>
      <property name="digitOffset">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="QLabel" name="labelTemperature">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Temperature</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelHumidity">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>60</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Humidity Ratio</string>
      </property>
     </widget>
     <widget class="ThermoMeter" name="humidity">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="toolTip">
       <string>Shows the temperature</string>
      </property>
      <property name="whatsThis">
       <string>The bar meter widget displays the pressure attached to it</string>
      </property>
      <property name="minimum">
       <double>20.000000000000000</double>
      </property>
      <property name="maximum">
       <double>80.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
      <property name="suffix">
       <string> [%]</string>
      </property>
      <property name="valueOffset">
       <double>276.000000000000000</double>
      </property>
      <property name="digitOffset">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupMoisture">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>231</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Moisture</string>
     </property>
     <widget class="ThermoMeter" name="moisture">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="toolTip">
       <string>Shows the temperature</string>
      </property>
      <property name="whatsThis">
       <string>The bar meter widget displays the pressure attached to it</string>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
      <property name="suffix">
       <string> [%]</string>
      </property>
      <property name="valueOffset">
       <double>275.000000000000000</double>
      </property>
      <property name="digitOffset">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupWaterLevel">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>50</y>
       <width>231</width>
       <height>321</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Water Level</string>
     </property>
     <widget class="ThermoMeter" name="waterLevel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>201</width>
        <height>201</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="toolTip">
       <string>Shows the temperature</string>
      </property>
      <property name="whatsThis">
       <string>The bar meter widget displays the pressure attached to it</string>
      </property>
      <property name="minimum">
       <double>0.000000000000000</double>
      </property>
      <property name="maximum">
       <double>100.000000000000000</double>
      </property>
      <property name="value">
       <double>0.000000000000000</double>
      </property>
      <property name="suffix">
       <string> [%]</string>
      </property>
      <property name="valueOffset">
       <double>275.000000000000000</double>
      </property>
      <property name="digitOffset">
       <double>10.000000000000000</double>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="freqLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>400</y>
       <width>241</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Measurement Frequency:</string>
     </property>
    </widget>
    <widget class="QwtCounter" name="freq">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>400</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>1.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="labelMinutesWaterLEvel">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>410</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Minutes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="ambienTab">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <attribute name="title">
     <string>Ambient Graph</string>
    </attribute>
    <widget class="QwtPlot" name="ambientGraph">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>871</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="moistureTab">
    <attribute name="title">
     <string>Moisture Graph</string>
    </attribute>
    <widget class="QwtPlot" name="moistureGraph">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>871</width>
       <height>431</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="waterTab">
    <attribute name="title">
     <string>Watering</string>
    </attribute>
    <widget class="QPushButton" name="watterButton">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>100</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Water</string>
     </property>
    </widget>
    <widget class="QPushButton" name="waterParamsFix">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>320</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set water parameters</string>
     </property>
    </widget>
    <widget class="QwtTextLabel" name="waterTimeLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>201</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="indent">
      <number>6</number>
     </property>
     <property name="plainText">
      <string>Water over time:</string>
     </property>
    </widget>
    <widget class="QwtTextLabel" name="waterParamsLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>170</y>
       <width>251</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="indent">
      <number>6</number>
     </property>
     <property name="plainText">
      <string>Water with thresholds: </string>
     </property>
    </widget>
    <widget class="QwtSlider" name="waterTimeSlider">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>431</width>
       <height>61</height>
      </rect>
     </property>
     <property name="upperBound">
      <double>60.000000000000000</double>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QwtSlider" name="waterTimeSlider_2">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>71</width>
       <height>191</height>
      </rect>
     </property>
     <property name="lowerBound">
      <double>5.000000000000000</double>
     </property>
     <property name="upperBound">
      <double>60.000000000000000</double>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QwtSlider" name="waterTimeSlider_3">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>260</y>
       <width>71</width>
       <height>191</height>
      </rect>
     </property>
     <property name="lowerBound">
      <double>0.000000000000000</double>
     </property>
     <property name="upperBound">
      <double>100.000000000000000</double>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QwtTextLabel" name="temperatureLabel2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="indent">
      <number>6</number>
     </property>
     <property name="plainText">
      <string>Temperature:</string>
     </property>
    </widget>
    <widget class="QwtTextLabel" name="soilMostLabel">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>310</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="indent">
      <number>6</number>
     </property>
     <property name="plainText">
      <string>Soil Moisture:</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QSlider" name="measurmentSwitch">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>10</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QwtTextLabel" name="startLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>0</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string>Measurements: </string>
   </property>
  </widget>
  <widget class="QwtTextLabel" name="onLabel">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string>On</string>
   </property>
  </widget>
  <widget class="QwtTextLabel" name="offLabel">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>40</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="plainText">
    <string>Off</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ThermoMeter</class>
   <extends>QWidget</extends>
   <header>analogwidgets/thermometer.h</header>
  </customwidget>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
  <customwidget>
   <class>QwtCounter</class>
   <extends>QWidget</extends>
   <header>qwt_counter.h</header>
  </customwidget>
  <customwidget>
   <class>QwtSlider</class>
   <extends>QWidget</extends>
   <header>qwt_slider.h</header>
  </customwidget>
  <customwidget>
   <class>QwtTextLabel</class>
   <extends>QFrame</extends>
   <header>qwt_text_label.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
